xml/zypper_upgrade.xml

```diff
@@ -67,119 +67,149 @@
    <step>
     <para>
      Run the online update to make sure the software management stack is
-     up-to-date. For more information, see
-     <xref linkend="cha-onlineupdate-you"/>.
+     up-to-date:
     </para>
+    <screen>&prompt.sudo;zypper patch</screen>
    </step>
    <step>
     <para>
-     Configure the repositories you want to use as update sources. Getting
-     this right is crucial. Either use &yast; (see
-     <xref linkend="sec-yast-software-instsource"/>) or <command>zypper</command>
-     (see <xref linkend="sec-zypper"/>).
-     The name of the repositories used in the following steps may vary
-     depending on your customizations.
+     Configure the repositories you want to use as update sources. Getting this
+     right is crucial. Either use &yast; (see <xref
+     linkend="sec-yast-software-instsource"/>) or <command>zypper</command>
+     (see <xref linkend="sec-zypper"/>) as used here.  The name of the
+     repositories used in the following steps may vary depending on your
+     customizations.
     </para>
     <para>
      To view your current repositories enter:
     </para>
-<screen>&prompt.user;zypper lr -u</screen>
-    <substeps performance="required">
+    <screen>&prompt.user;zypper --releasever=15.3 lr -u</screen>
```

Comment:
* I don't think I understand why you added `--releasever=15.3` here.
* Indenting screens like this only means that the next person that comes along using xmlformat, will reset the indent. This is Sisyphean work and will only create repo churn.